src/pages/components/leadspace.mdx

```diff
@@ -107,70 +130,109 @@ There are two expressive type styles to choose from when using Lead space left-a
+
+There are three recommended expressive type styles to choose from when using Lead space centered:
+
+- `$expressive-heading-06` is recommended for pages that surface detailed, productive content to users such as product pages.
+- `$display-01` and `$expressive-heading-05` is recommended for pages that have rich editorial, expressive moments such as marketing or campaign pages.
+
+![Lead space with $expressive-heading-06](../../images/pattern/leadspace/leadspace_typestyles_centered_demo_1.jpg)
+
+<Caption>
+  $expressive-heading-06 can be used to highlight the name of the product,
+  offering or title of topic. This option is optimal if using all Lead space and
+  components together. 
+</Caption>
+
+<br />
+
+![Lead space with $display-01](../../images/pattern/leadspace/leadspace_typestyles_centered_demo_2.jpg)
+
+<Caption>
+  $display-01 can be used to celebrate strong, editorial messaging.
+</Caption>
+
+## Design and functional specifications
+
+The design specs and functional specs for Lead space can be viewed <a href="https://ibm.box.com/s/d9jey3rlfxqmxu059f0x9dtcx7f5ehuh" target="_blank" rel="noreferrer">here</a>.
 
-![$expressive-heading-06 at 1312 xlg breakpoint](../../images/pattern/leadspace/LeadSpace_Left_Heading06_Specs.jpg)
+### Lead space left-aligned
+
+![$expressive-heading-06 at 1312 xlg breakpoint](../../images/pattern/leadspace/leadspace_specs_left_demo_1.jpg)
 
-<Caption>$expressive-heading-06 at 1312 xlg breakpoint</Caption>
+<Caption>Example: $expressive-heading-06 at xlg breakpoint (1312px) for a Medium (480px height) lead space</Caption>
 
 <br />
 
-1. **Heading:** type: $expressive-heading-06, color: $text-01, width: 95% of container (8 columns)
-1. **Content spacer:** Minimum space between lead space heading and short copy: 96px
-1. **Short copy:** \*optional, type: $expressive-heading-03, color: $text-01, width: 95% of container (4 columns)
-1. **Button group:** Use the Button group component
-1. **Gradient stop:** color: black (#000000), opacity: 75%, linear: left to right, width: 50% of total container width
-1. **Background image:** \*optional, width: 100% of container for all breakpoints. For detailed guidelines, visit [the imagery section on the Lead space pattern page](/patterns/lead-space#imagery).
+1. **Heading** <br/>Type: $expressive-heading-06, color: $text-01, width: calc(8col – 32px)
+1. **Content spacer** <br/>Minimum space between lead space heading and short copy: 96px
+1. **Short copy** \*optional <br/>Type: $expressive-heading-03, color: $text-01, width: calc(6col – 32px)
+1. **Button group** <br/>Use the Button group component
+1. **Gradient (applied over image)** <br/>Color: black (#000000), linear: left to right <br/>Opacity: 1 at start, 0.4 at 50%, 0 at 75% of total container width
+1. **Background image** \*optional <br/>Cover, width: 100% of container for all breakpoints. For detailed guidelines, visit [the imagery section on the Lead space pattern page](/patterns/lead-space#imagery).
```

Comment:
I see why you aded the <br /> , esp. on Gradient line where there are 2 line items... I suspect there's a deeper IA problem though. If we wanna document specs, there should probably be a better way to do it. It's honestly also difficult to keep up to date. Nevertheless, this looks good.